test/circular/main.js

```diff
@@ -0,0 +1,6 @@
+import { getBar } from './foo';
+import { getFoo } from './bar';
+
+// true in dev, false in production
```

Comment:
What do you mean with that comment?
In the multibuild_test you check against true in prod.html